packages/client/tests/functional/17810-direct-url-missing/prisma/_schema.ts

```diff
@@ -1,20 +1,19 @@
+import { computeSchemaHeader } from '../../_utils/computeSchemaHeader'
 import { idForProvider } from '../../_utils/idForProvider'
 import testMatrix from '../_matrix'
 
-export default testMatrix.setupSchema(({ provider }) => {
+export default testMatrix.setupSchema(({ provider, providerFlavor }): string => {
+  const schemaHeader = computeSchemaHeader({
+    provider,
+    providerFlavor,
+    directUrl: `env("DIRECT_DATABASE_URI_${providerFlavor ? providerFlavor : provider}")`,
```

Comment:
Maybe we can make this a boolean instead?

Reply:
It's only used in one place at the moment. Maybe can be done later.